source/administration/backup.rst

```diff
@@ -25,6 +25,8 @@ To backup your Mattermost server:
      - If you use local storage using a non-default directory specified in the ``Directory`` setting in ``config.json``, back up files in that location.
      - If you store your files in S3, you can typically keep the files where they are located without backup.
      
+Please note that to make a 'clean' backup you need to stop mattermost during the duration of the backup otherwise the database and files may become out of sync.
```

Comment:
```suggestion
Please note that to make a 'clean' backup you need to stop Mattermost during the duration of the backup otherwise the database and files may become out of sync.
```